server/paymentProviders/utils.js

```diff
@@ -4,6 +4,7 @@ import activities from '../constants/activities';
 import { CollectiveType } from '../constants/collectives';
 import ExpenseStatus from '../constants/expense-status';
 import ExpenseType from '../constants/expense-type';
+import { PAYMENT_METHOD_SERVICE, PAYMENT_METHOD_TYPE } from '../constants/paymentMethods';
```

Comment:
We should merge this file into `stripe/virtual-cards.ts` since it only contains functions relevant to stripe issuing.